services/apps/src/apps/routes/admin.clj

```diff
@@ -153,6 +156,58 @@
           on the fields included in the request."
           (ok (apps/admin-update-category current-user (assoc body :id category-id)))))
 
+(defroutes* admin-ontologies
+
+  (GET* "/" []
+        :query [params SecuredQueryParams]
+        :return ActiveOntologyDetailsList
+        :summary "List Ontology Details"
+        :description
+"Lists Ontology details saved in the metadata service.
+
+#### Delegates to metadata service
+    GET /ontologies"
```

Comment:
I thought about that too, but wasn't sure if just using this `HEREDOC` style of string was more readable than `str` concatenation with a helper-function in the middle.